lib/json_web_token.rb

```diff
@@ -0,0 +1,9 @@
+class JsonWebToken
+  def self.issue(payload)
```

Comment:
Why this method called issue? It will be more readable to use encode instead.

Reply:
Updated